sim/src/sim/mod.rs

```diff
@@ -211,11 +211,7 @@ impl Sim {
             transit: TransitSimState::new(map),
             cap: CapSimState::new(map, &opts),
             trips: TripManager::new(),
-            pandemic: if let Some(rng) = opts.enable_pandemic_model {
-                Some(PandemicModel::new(rng))
-            } else {
-                None
-            },
+            pandemic: opts.enable_pandemic_model.map(PandemicModel::new),
```

Comment:
`bool` implements `map`?! Mind blown. Makes so much sense.

Reply:
I think `enable_pandemic_model` is an `Option` here?

`bool` does implement `then` since 1.50 though, which is similar.